### PR TITLE
feat(controls): Create new rotate control + add to image viewer

### PR DESCRIPTION
**Pull request by cweeii**

### CODE REVIEW

File: src/lib/viewers/controls/rotate/RotateControl.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import IconRotate24 from '../icons/IconRotate24';
+import './RotateControl.scss';
+
+export type Props = {
+    onRotateLeft: () => void;
+};
+
+export default function RotateControl({ onRotateLeft }: Props): JSX.Element {
+    return (
+        <div className="bp-RotateControl">
```
Comment (jstoffan) [inline src/lib/viewers/controls/rotate/RotateControl.tsx:11]: I don't think this needs to be in its own group. Can just be the button on its own.

File: src/lib/viewers/controls/rotate/RotateControl.scss [hunk 2]
```diff
@@ -0,0 +1,19 @@
+@import '../styles';
+
+.bp-RotateControl {
+    @include bp-ControlGroup;
+}
+
+.bp-RotateControl-button {
+    @include bp-ControlButton($width: 32px);
```
Comment (jstoffan) [inline src/lib/viewers/controls/rotate/RotateControl.scss:8]: Is this the width of the button in the current experience?

File: src/lib/viewers/controls/rotate/RotateControl.scss [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+}
+
+.bp-RotateControl-current {
```
Comment (jstoffan) [inline src/lib/viewers/controls/rotate/RotateControl.scss:11]: Is this used or was this copy/pasted from elsewhere?

File: src/lib/viewers/controls/rotate/RotateControl.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+            <button
+                className="bp-RotateControl-button"
+                data-testid="bp-RotateControl-out"
```
Comment (jstoffan) [inline src/lib/viewers/controls/rotate/RotateControl.tsx:14]: What is `-out` here?

File: src/lib/viewers/controls/rotate/RotateControl.tsx (hunk 1 above)
Comment (cweeii) [reply src/lib/viewers/controls/rotate/RotateControl.tsx:11]: Okay, will remove!

File: src/lib/viewers/controls/rotate/RotateControl.scss (hunk 2 above)
Comment (cweeii) [reply src/lib/viewers/controls/rotate/RotateControl.scss:8]: I'm going to check it visually and see the difference.

File: src/lib/viewers/controls/rotate/RotateControl.scss (hunk 3 above)
Comment (cweeii) [reply src/lib/viewers/controls/rotate/RotateControl.scss:11]: This was copied from ZoomControls, will remove!

File: src/lib/viewers/controls/rotate/RotateControl.tsx (hunk 4 above)
Comment (cweeii) [reply src/lib/viewers/controls/rotate/RotateControl.tsx:14]: This was copied from ZoomControls, will remove!

File: src/lib/viewers/controls/rotate/__tests__/RotateControl-test.tsx [hunk 5]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import RotateControl from '../RotateControl';
+import IconRotate24 from '../../icons/IconRotate24';
```
Comment (ConradJChan) [inline src/lib/viewers/controls/rotate/__tests__/RotateControl-test.tsx:4]: By convention our import order is: absolute imports > relative imports > stylesheets -- with each category alphabetized

File: src/lib/viewers/controls/rotate/RotateControl.scss [hunk 6]
```diff
@@ -0,0 +1,5 @@
+@import '../styles';
+
+.bp-RotateControl {
+    @include bp-ControlButton();
```
Comment (jstoffan) [inline src/lib/viewers/controls/rotate/RotateControl.scss:4]: We can leave the `()` off the end of a standard import.

File: src/lib/viewers/controls/rotate/__tests__/RotateControl-test.tsx (hunk 5 above)
Comment (cweeii) [reply src/lib/viewers/controls/rotate/__tests__/RotateControl-test.tsx:4]: Got it, thanks!

File: src/lib/viewers/controls/rotate/RotateControl.scss (hunk 6 above)
Comment (cweeii) [reply src/lib/viewers/controls/rotate/RotateControl.scss:4]: Got it, will change it!

**Review by jstoffan — APPROVED** [review]

**Pull request merged**